Gang Mode Fixes

Ikarrus opened the pull request:
- Fixed needing to wait for the next payday before you can tag a territory that has it's old tag cleaned off.
- Added distinct lieutenant ranks in gang messages
- Fixed lieutenants being able to do what only bosses should be able to do (Promote & recall)

Ikarrus committed:
Territory Tagging Fix

Fixed needing to wait for the next payday before you can tag a territory that has it's old tag cleaned off.

Ikarrus committed:
no message

francinum commented:
:+1:

Dorsisdwarf commented:
Can you fix the problem that tags are situated on walls, instead of on the tile you were standing on (Like other wall objects), thus allowing them to tag through walls, and for tags to be seen through walls.

Miauw62 commented:
that is intended.

Ikarrus committed:
Added distinct lieutenant ranks
Fixed lieutenants being able to do what only bosses should do (Promote, recall)
